RELEASE-NOTES.txt

```diff
@@ -1,3 +1,7 @@
+15.9
+-----
+ * [internal] Removed feature flag for unified Site Address views. [#14954]
```

Comment:
Please forgive me if this is wrong, but my understanding was that the release notes are relevant for testers / users.  Isn't the removal of the feature flag completely transparent for them?

Just wondering.

Reply:
We can flag changes with `[internal]` if something _should_ be transparent but could cause regressions. These aren't publicized, but are noted in the call for testing posts to verify nothing has changed.